package.json

```diff
@@ -57,8 +57,8 @@
     "ssb-keys": "^8.1.0",
     "ssb-ref": "^2.14.3",
     "ssb-validate": "^4.1.1",
+    "tap-arc": "^0.3.4",
```

Comment:
I was battling a bit with the Update index test that uses the `status` call and updated this while tested. Locally that test seems to randomly fail, might be related to our test runner, but still it shouldn't matter in what order the tests are run.